ios/FluentUI.Demo/FluentUI.Demo/ReadmeStrings.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+class ReadmeStrings {
+
+    static let segmentedControlString: String? = "A segmented control lets someone select one option from a set of two or more segments in a single, horizontal container.\n\nSegmented controls work well for changing states of elements or views within a single context, like filtering search results. It’s best not to use them to initiate actions or navigate to a new page. To let people navigate between the main sections of an app, use the tab bar."
+
+    static let dateTimePickerString: String? = "A date and time picker is a specialized interface that lets people choose a single date, a range of dates, or a time.\n\nDate and time pickers provide context that can help people make decisions, like the day of the week a date falls or unavailable dates. Avoid using date and time pickers for dates people know well or that will require a lot of scrolling, like birthdays."
+
+    static let badgeString: String? = "A badge is a compact, interactive, textual representation of a person. It is generally a representation of user-input text that maps to an entry in a database."
+
+    static let buttonString: String? = "A button triggers a single action or event.\n\nUse buttons for important actions like submitting a response, committing a change, or moving to the next step. If you need to navigate to another place, try a link instead."
+}
```

Comment:
Was wondering where these strings are coming from?

Reply:
might be better off if these strings just lve in the democontroller itself?

Reply:
> Was wondering where these strings are coming from?

Strings are from a figma Harrie shared with me. I'll link it to you on teams.

Reply:
> might be better off if these strings just lve in the democontroller itself?

Some strings can be quite long I thought it'd be a good idea to abstract them away from the demo controllers. But I can go ahead and make the change.